Centaurus.Domain/accounts/AccountStorage.cs

```diff
@@ -9,8 +9,8 @@ namespace Centaurus.Domain
 {
     public class AccountStorage
     {
-        public AccountStorage(IEnumerable<Account> accounts)
-            :this(accounts.Select(a => new AccountWrapper(a)))
+        public AccountStorage(IEnumerable<Account> accounts, RequestRateLimits defaultRequestRateLimits)
```

Comment:
Not sure that this will be faster than accessing `Global.Constellation.RequestRateLimits` in this particular case.

Reply:
We need it in case of reverting constellation state to some specific Apex. So we can't access Global.Constellation.RequestRateLimits, we need to pass settings for this Apex.

Reply:
👍